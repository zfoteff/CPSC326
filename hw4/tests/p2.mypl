
# Hello World program
fun int main(
)
print
(
         "Hello World!\n"

      ) end
       
       
