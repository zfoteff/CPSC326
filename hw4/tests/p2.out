fun int main()
   print("Hello World!\n")
end

